app/src/main/java/com/globant/tutoria/subscriberssearch/fragment/RecyclerViewFragment.java

```diff
@@ -0,0 +1,98 @@
+        LINEAR_LAYOUT_MANAGER
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View rootView = inflater.inflate(R.layout.recycler_view_fragment, container, false);
+        rootView.setTag(TAG);
+        recyclerViewSubscribers = (RecyclerView) rootView.findViewById(R.id.recycler_view_subscribers);
+        mCurrentLayoutManagerType = LayoutManagerType.LINEAR_LAYOUT_MANAGER;
+        if (savedInstanceState != null) {
+            // Restore saved layout manager type.
+            mCurrentLayoutManagerType = (LayoutManagerType) savedInstanceState
+                    .getSerializable(KEY_LAYOUT_MANAGER);
+        }
+        setRecyclerViewLayoutManager(mCurrentLayoutManagerType);
+        OctokitInterface octokitInterface = RetrofitClient.getClient().create(OctokitInterface.class);
+        Call<List<SubscribersModel>> callService = octokitInterface.getSubscribers();
+        callService.enqueue(new Callback<List<SubscribersModel>>() {
+            @Override
+            public void onResponse(Call<List<SubscribersModel>> call, Response<List<SubscribersModel>> response) {
+                List<SubscribersModel> listSubscribers = response.body();
+                RecyclerView.Adapter adapter = new DataSubscribersAdapter(listSubscribers);
+                recyclerViewSubscribers.setAdapter(adapter);
+            }
+            @Override
+            public void onFailure(Call<List<SubscribersModel>> call, Throwable t) {
+                Log.e("onFailure", t.getMessage());
+            }
+        });
+        return rootView;
+    }
+
+    public void setRecyclerViewLayoutManager(LayoutManagerType layoutManagerType) {
+        int scrollPosition = 0;
+        if (recyclerViewSubscribers.getLayoutManager() != null) {
+            scrollPosition = ((LinearLayoutManager) recyclerViewSubscribers.getLayoutManager())
+                    .findFirstCompletelyVisibleItemPosition();
+        }
+        switch (layoutManagerType) {
+            case GRID_LAYOUT_MANAGER:
+                mLayoutManager = new GridLayoutManager(getActivity(), SPAN_COUNT);
+                mCurrentLayoutManagerType = LayoutManagerType.GRID_LAYOUT_MANAGER;
+                break;
+            case LINEAR_LAYOUT_MANAGER:
+                mLayoutManager = new LinearLayoutManager(getActivity());
+                mCurrentLayoutManagerType = LayoutManagerType.LINEAR_LAYOUT_MANAGER;
+                break;
+            default:
+                mLayoutManager = new LinearLayoutManager(getActivity());
+                mCurrentLayoutManagerType = LayoutManagerType.LINEAR_LAYOUT_MANAGER;
+        }
+        recyclerViewSubscribers.setLayoutManager(mLayoutManager);
+        recyclerViewSubscribers.scrollToPosition(scrollPosition);
```

Comment:
This will always scroll to first position because the layoutManagerType will always be null